LondonBritishLibrary/orient/BLorient11763.xml

```diff
@@ -0,0 +1,125 @@
+                            <title ref="LIT7057Fever"/>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎተ፡ ፌራ፡ ወንዳድ፡ ዘቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ <gap reason="ellipsis"/>
+                                ሌኬ፡ ኤንካ፡ ጌሌ፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                        <msItem xml:id="ms_i3">
+                            <title ref="LIT00000000000000000000000">Prayer against ʿaynat šimdān</title>
+                            <incipit xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> ጸሎት፡ በእንተ፡ ሕማመ፡ ዓይነት፡ ሺምዳን፡ ተክለ፡ ሽርታን፡</incipit>
+                            <textLang mainLang="gez"/>
+                        </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Scroll">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="panel">1</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>715</height>
+                                        <width>97</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc> 
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>Mediocre, square handwriting.</desc>
+                                <date notBefore="1600" notAfter="1699" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <desc>One coloured magical picture.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="true">
+                                <decoNote xml:id="b1" type="SlipCase"><material key="leather"/>Contained in a cylindrical leather case.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1600" notAfter="1699">17th century</origDate>
+                        </origin>
+                        <provenance>Two owners are named: <persName role="owner" ref="PRS14519WaldaSamuel">Walda Sāmuʾel</persName>
+                            and <persName role="owner" ref="PRS14520BakweraSeyon">Bakʷǝra Ṣǝyon</persName></provenance>
```

Comment:
Bakʷra without schwa, no?

Reply:
OK, I will also change it in PRS14520BakweraSeyon.